### PR TITLE
Update puns.ts

### PR DESCRIPTION
**Pull request by markkramerus**

### CODE REVIEW

File: src/utils/puns.ts [hunk 1]
```diff
@@ -19,7 +19,24 @@ const CLEAN_RUN_PUNS = [
...
+  "You've earned a seal of approval.",
+  "You're swimming in success!",
+  "You're making waves now!",
+  "You've turned the tide.",
+  "You're catching on now!",
+  "You're in the swim of things now!",
+  "Catch of the day, right here!",
+  "That was a big fish to fry!",
+  "Nice job! Are you hooked yet?",
+  "You've found your sea legs!",
```
Comment (cmoesel) [inline src/utils/puns.ts:34]: This is the same as line 23 above (except `!` instead of `.`).

File: src/utils/puns.ts [hunk 2]
```diff
@@ -34,16 +51,26 @@ const WARNING_PUNS = [
...
+  'Let minnow if you need some help',
+  "You're kraken me up.",
+  "Don't be koi about asking for help.",
+  "You seem a bit rudderless.",
+  'Keep swimming, Dory.',
+  "You should gut it out.",
+  "Don't get stuck in the doldrums.",
+  "Are you in a no-wind situation?",
+  "Sailors, take warning!",
+  "You seem to be casting about.",
```
Comment (cmoesel) [inline src/utils/puns.ts:64]: This is very similar to line 46.

File: src/utils/puns.ts [hunk 3]
```diff
@@ -34,16 +51,26 @@ const WARNING_PUNS = [
...
+  "This looks a bit fishy."
```
Comment (cmoesel) [inline src/utils/puns.ts:65]: This is very similar to line 44.

File: src/utils/puns.ts [hunk 4]
```diff
@@ -56,9 +83,22 @@ const ERROR_PUNS = [
   'You are battering FSH',
   'Swim back to School of FSH',
   'You need to mullet over',
+  "You're krilling me!",
+  "You're fluking it up!",
```
Comment (cmoesel) [inline src/utils/puns.ts:87]: Might this offend some of our more sensitive users?

File: src/utils/puns.ts [hunk 5]
```diff
@@ -56,9 +83,22 @@ const ERROR_PUNS = [
   'You are battering FSH',
   'Swim back to School of FSH',
   'You need to mullet over',
```
Comment (cmoesel) [inline src/utils/puns.ts:85]: I just noticed we also have `You should mullet over` in the warnings section. Maybe remove this one?

**Review by cmoesel — CHANGES_REQUESTED** [review]
You, sir, are prolific.

I found a few duplicates...

File: src/utils/puns.ts (hunk 4 above)
Comment (markkramerus) [reply src/utils/puns.ts:87]: I thought it was kind of edgy... I'll cut it out.

File: src/utils/puns.ts (hunk 1 above)
Comment (markkramerus) [reply src/utils/puns.ts:34]: cut line 34

File: src/utils/puns.ts (hunk 2 above)
Comment (markkramerus) [reply src/utils/puns.ts:64]: Cut line 46

File: src/utils/puns.ts (hunk 5 above)
Comment (markkramerus) [reply src/utils/puns.ts:85]: Removed.

File: src/utils/puns.ts [hunk 6]
```diff
@@ -56,9 +83,22 @@ const ERROR_PUNS = [
   'You are battering FSH',
```
Comment (mint-thompson) [inline src/utils/puns.ts:83]: I've wanted to remove this one for awhile. Could we get rid of it?

**Review by mint-thompson — CHANGES_REQUESTED** [review]

File: src/utils/puns.ts (hunk 6 above)
Comment (markkramerus) [reply src/utils/puns.ts:81]: No objection here. Let me do one more push.

**Review by cmoesel — COMMENTED** [review]
This is failing CI because strings must be enclosed in single-quotes (not double-quotes). The only exception to this rule is if the string contains a single-quote, _then_ it can be wrapped in double-quotes.

The easiest way to fix this is just to run `npm run lint:fix` and the linter will fix it for you. Then add, commit, and push.

**Review by cmoesel — APPROVED** [review]
Thanks, @markkramerus. I expect our users will appreciate this more than any other feature we've released in the last six months.

**Review by mint-thompson — APPROVED** [review]

**Pull request merged**